### PR TITLE
Update Creation.rst

### PR DESCRIPTION
**Pull request by ayacoo**

### CODE REVIEW

File: Documentation/ApiOverview/UpdateWizards/Creation.rst [hunk 1]
```diff
@@ -7,69 +7,114 @@ Creating generic update wizards
...
-Method :php:`performUpdate`
+Method :php:`executeUpdate`
    Is called if the user triggers the wizard. This method should contain, or call,
-   the code that is needed to execute the update. :php:`$databaseQueries` and
-   :php:`$customMessage` can be used as reference to provide further information to
-   the user after the update function is completed. :php:`$databaseQueries` has to
-   be an array, where each value is a string containing the query. This array should
-   only contain executed queries. :php:`$customMessage` is a string, where further
-   information is provided for the user after the updated process has completed.
+   the code that is needed to execute the update.
```
Comment (mbrodala) [inline Documentation/ApiOverview/UpdateWizards/Creation.rst]: At least here an explanation about the boolean return value should be added.

File: Documentation/ApiOverview/UpdateWizards/Creation.rst [hunk 2]
```diff
@@ -7,69 +7,114 @@ Creating generic update wizards
...
-   only contain executed queries. :php:`$customMessage` is a string, where further
-   information is provided for the user after the updated process has completed.
+   the code that is needed to execute the update. 
+
+Method :php:`updateNecessary`
+   Is called to check whether the updater has to run. Therefore a boolean has to be
+   returned.
+   
+Method :php:`getPrerequisites`
+   Returns an array of class names of Prerequisite classes. This way a wizard can
```
Comment (mbrodala) [inline Documentation/ApiOverview/UpdateWizards/Creation.rst]: Why uppercase `P`? Any maybe add some specific example here or in the code above.

File: Documentation/ApiOverview/UpdateWizards/Creation.rst [hunk 3]
```diff
@@ -7,69 +7,114 @@ Creating generic update wizards
...
+   Returns an array of class names of Prerequisite classes. This way a wizard can 
+   define dependencies like "database up-to-date" or "reference index updated".
 
 Marking wizard as done
 ======================
 
 As soon as the wizard has completely finished, e.g. it detected that no update is
 necessary anymore, or that all updates were completed successfully, the wizard should
-be marked as done. To mark the wizard as done, call :php:`$this->markWizardAsDone`.
+be marked as done. To mark the wizard as done, call :php:`updateNecessary`.
```
Comment (mbrodala) [inline Documentation/ApiOverview/UpdateWizards/Creation.rst]: Actually no additional call is required anymore here.

Comment (ayacoo) [reply Documentation/ApiOverview/UpdateWizards/Creation.rst]: So we could remove this section completely? Or would you like to have a note that "wizard done" now gets the info from the function executeUpdate?

File: Documentation/ApiOverview/UpdateWizards/Creation.rst (hunk 2 above)
Comment (ayacoo) [reply Documentation/ApiOverview/UpdateWizards/Creation.rst]: Copy paste from functions header ;-) Adding an example is a good idea. Thanks!

**Comment by mbrodala** [comment]
Thanks for the update!

**Pull request merged**